package.json

```diff
@@ -99,7 +99,7 @@
     "fast-deep-equal": "1.0.0",
     "object-assign": "4.1.1",
     "prop-types": "15.6.0",
-    "react-cursor-position": "2.4.1",
+    "react-cursor-position": "3.0.2",
```

Comment:
The 3.x version of react-cursor-position is not yet compatible with react-image-magnify. Please install react-cursor-position 2.5.1 by running the command `yarn add react-cursor-position@2.5.1`. Thanks!